templates/repository/common/CONTRIBUTING.md

```diff
@@ -93,8 +94,8 @@ to help out:
   look at discussions in the forum and take part in community events. More info
   on this in [Communication](#communication).
 
-- **Answer discussions.** There are at all times a number of unanswered
-  discussions on GitHub, you can see an
+- **Answer discussions.** There are several unanswered discussions at all times
+  on GitHub, you can see an
```

Comment:
What do you (and your grammar tools) think about rephrasing it like this?

```suggestion
- **Answer discussions.** At all times, there are several unanswered discussions
  on GitHub. You can see an
```

Reply:
Sounds good.